server/src/test/java/org/elasticsearch/index/codec/tsdb/TsdbDocValueBwcTests.java

```diff
@@ -42,6 +42,7 @@
 import org.elasticsearch.index.codec.perfield.XPerFieldDocValuesFormat;
 import org.elasticsearch.index.codec.tsdb.ES87TSDBDocValuesFormatTests.TestES87TSDBDocValuesFormat;
 import org.elasticsearch.index.codec.tsdb.es819.ES819TSDBDocValuesFormat;
+import org.elasticsearch.index.codec.tsdb.es819.ES819TSDBDocValuesFormatTests;
 import org.elasticsearch.test.ESTestCase;
 import org.hamcrest.Matchers;
 
```

Comment:
The `testMixedIndex()` needs to be update to use binary doc values similar to `ES819TSDBDocValuesFormatTests`. And then we need to test against codec version 0 to version 1.